### PR TITLE
check in README2

### PR DESCRIPTION
**Pull request by leerho**
The PR name is a misnomer, this includes description updates to the KLL SQLX files as well as README2

### CODE REVIEW

File: kll/sqlx/kll_sketch_float_get_rank.sqlx [hunk 1]
```diff
@@ -24,12 +24,19 @@ RETURNS FLOAT64
 LANGUAGE js
 OPTIONS (
   library=["gs://$GCS_BUCKET/kll_sketch.js"],
-  description = '''Returns an approximation to the normalized rank of the given value. from 0 to 1, inclusive.
+  description = '''Returns an approximation to the normalized rank, on the interval (0.0, 1.0], of the given value.
```
Comment (AlexanderSaydakov) [inline kll/sqlx/kll_sketch_float_get_rank.sqlx]: why zero is not included? the rank can be zero

File: kll/sqlx/kll_sketch_float_get_cdf.sqlx [hunk 2]
```diff
@@ -24,20 +24,36 @@ RETURNS ARRAY<FLOAT64>
...
+  The end of each interval is the associated split-point except for the top interval
+  where the end is the maximum input value of the stream.
+
+Param inclusive: if true and the upper boundary of an interval equals a value retained by the sketch, the interval will include that value. 
+  If the lower boundary of an interval equals a value retained by the sketch, the interval will exclude that value.
+
+  If false and the upper boundary of an interval equals a value retained by the sketch, the interval will exclude that value. 
+  If the lower boundary of an interval equals a value retained by the sketch, the interval will include that value.
+
+Returns: the CDF as a monotonically increasing FLOAT64 array of M+1 cumulative probablities on the interval (0.0, 1.0].
```
Comment (AlexanderSaydakov) [inline kll/sqlx/kll_sketch_float_get_cdf.sqlx]: zero is included, isn't it?

File: kll/sqlx/kll_sketch_float_get_rank.sqlx (hunk 1 above)
Comment (leerho) [reply kll/sqlx/kll_sketch_float_get_rank.sqlx]: Oops.  Copy/paste error.

File: kll/sqlx/kll_sketch_float_get_cdf.sqlx (hunk 2 above)
Comment (leerho) [reply kll/sqlx/kll_sketch_float_get_cdf.sqlx]: Yes, will fix.

**Review by AlexanderSaydakov — APPROVED** [review]

**Pull request merged**